editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -115,6 +116,7 @@ const resizeStrategies: MetaCanvasStrategy = (
       flexResizeStrategy,
       basicResizeStrategy,
       resizeGridStrategy,
+      gridResizeElementStrategy,
     ],
   )
 }
```

Comment:
would you mind adding the new resize strategy to `isResizableStrategy` as well (line 524)?

Reply:
https://github.com/concrete-utopia/utopia/pull/6038/commits/c471b7ac68bee6609434bb3c058fa86c13fec73d